lib/fly.js

```diff
@@ -85,12 +85,18 @@ module.exports = Fly
  * @return                      The current Fly instance.
  */
 Fly.prototype.source = function (globs) {
-	globs = arrify(globs)
+	globs = flatten([].slice.call(arguments))
```

Comment:
It's ignored because of how you're trying to pull out an `options` from a spread parameter (& use of `pop()`)

Previously: 

```javascript
function source(...globs) {
  console.log('1: ', globs);
  console.log('2: ', arrify(globs));
}

source('first', 'second');
//=> 1: ['first', 'second']
//=> 2: ['first', 'second']
```